src/comms.cpp

```diff
@@ -1139,6 +1128,25 @@ void comms_loop_sec1()
 
         clearToSend = false;
 
+        // this byte is received with invalid parity
+        // it is sent when there is no buss traffic (need to look at it with scope)
+        if (ser_byte == 0xFF)
+        {
+            syncByteCount++;
+            if (syncByteCount == 10)
+            {
+                syncByteCount = 0;
+                // alternate way to detect no wall panel
+                // not in use as of now
+                // but could start emulator here
```

Comment:
We have to be very careful about starting emulator outside of initial startup... we could get a series of 0xFF's if someone is doing maintenance on the GDO and temporarily disconnects the wall panel.  Also there is a reason why we wait 15 seconds at power up to conclude that there is no wall panel... apparently it can take some time after GDO power on for the wall panel to initialize itself and so if there is a power interruption and everything comes back at the same time... the ratgdo needs to wait a bit to be sure that the GDO and it's digital wall panel initialize before concluding that there is none.